crm_claim_rma/wizards/claim_make_picking.py

```diff
@@ -130,6 +130,13 @@ def _get_picking_data(self, claim, picking_type, partner_id):
         }
 
     def _get_picking_line_data(self, claim, picking, line):
+        move_id = None
+        if not line.origin_move:
+            moves = line.mapped('invoice_line_id.sale_line_ids.procurement_ids')
```

Comment:
With this `mapped` you'll get `procurement.order` records, please rename variable accordingly.